src/content/en/fundamentals/instant-and-offline/offline-storage-pwa/index.md

```diff
@@ -0,0 +1,260 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+
+{# wf_updated_on: 2016-09-29 #}
+{# wf_published_on: 2016-09-29 #}
+
+# **Offline Storage for Progressive Web Apps** {: .page-title }
+
+{% include "web/_shared/contributors/addyosmani.html" %}
```

Comment:
Please include yourself in the contributors list. I know it takes quite some time to get these posts correctly edited and formatted.